src/atomate2/common/phonon_utils.py

```diff
@@ -0,0 +1,31 @@
+"""Define utilities for the phonon workflow."""
+from phonopy.units import VaspToTHz
+
+from atomate2.aims.utils.units import omega_to_THz
+
+
+def get_factor(code: str):
+    """
+    Get the frequency conversion factor to THz for each code.
+
+    Parameters
+    ----------
+    code: str
+        The code to get the conversion factor for
+
+    Returns
+    -------
+    float
+        The correct conversion factor
+
+    Raises
+    ------
+    ValueError
+        If code is not defined
+    """
+    # TARP: This is based on self.code == "vasp" in the old forcefields workflow
+    if code in ["forcefields", "vasp"]:
```

Comment:
Is this correct? In the original forcefields workflow the code was still set to VASP so I assumed that the factors were the same.

Reply:
Yes, exactly.